ha/setup/ha_setup.py

```diff
@@ -125,6 +125,28 @@ def get_node_name(self):
         Log.info(f"Read node name: {node_name}")
         return node_name
 
+    @staticmethod
```

Comment:
How is this change related to post_upgrade?

Reply:
There is an existing _get_s3_instance() method at line #314 below.

Reply:
Its a private method of ConfigCmd class. And private methods, we can not access outside (though, python will allow this in our case). 2nd, now, this method is not just required at config phase, but it has use case at upgrade case too. 
And, the instance count will be same (static). Right ?
Hence, I made it static method of Cmd class. implementation wise, its same as _get_s3_instance().
Let me know if still change is needed. Will do it.